code/game/objects/items/clown_items.dm

```diff
@@ -123,6 +136,9 @@
 		clean_speedies = cleanspeed * min(user.mind.get_skill_modifier(/datum/skill/cleaning, SKILL_SPEED_MODIFIER)+0.1,1) //less scaling for soapies
 	//I couldn't feasibly  fix the overlay bugs caused by cleaning items we are wearing.
 	//So this is a workaround. This also makes more sense from an IC standpoint. ~Carn
+	if(uses <= 0)
+		to_chat(user, span_warning("No good, you need to recharge!"))
+		return
```

Comment:
Rather then putting this in `soap/afterattack`, I think it'd be better to override `afterattack` on `soap/cyborg`
That way you can do the uses check, and if we have uses left, call parent, otherwise send our warning